LEGO1/lego/legoomni/src/race/carrace.cpp

```diff
@@ -386,7 +386,7 @@ MxLong CarRace::HandleType0Notification(MxNotificationParam&)
 void CarRace::FUN_10017820(MxS32 p_param1, MxS16 p_param2)
 {
 	MxS32 local4;
-	MxStillPresenter* presenter;
+	MxStillPresenter* presenter = NULL;
```

Comment:
What's this change for (and the others like it?)

Reply:
without that, MSVC cries for potentially uninitialized variable